app/utils/validatePhoneNumber.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ *  @module normalize/validatePhoneNumber
+ * Validator for US only phone numbers.
+ */
+
+// eslint-disable-next-line import/no-unused-modules
```

Comment:
Please don't silence this lint rule. It's put there to prevent us from leaving around any code that is unused. If it goes off, it means that any publicly exported name isn't actually being used anywhere else in the app.